java/vector/src/main/java/org/apache/arrow/vector/complex/impl/StructOrListWriterImpl.java

```diff
@@ -88,7 +88,7 @@ public StructOrListWriter struct(final String name) {
    *
    * @param name Unused.
    */
-  public StructOrListWriter listoftstruct(final String name) {
+  public StructOrListWriter listofstruct(final String name) {
```

Comment:
Just curious, should it be?

```suggestion
  public StructOrListWriter listOfStruct(final String name) {
```

Reply:
That'd be my preference, yes. But I'm not sure how much changes people would tolerate, so I tend to make minimal suggested changes.

I'm willing to make this change if requested.